src/disemvowel.js

```diff
@@ -0,0 +1,4 @@
+export const disemvowel = (string) => {
```

Comment:
I think this would be a better version of the RegeExp. 
```js
export const disemvowel = string => 
  string.toLowerCase().replace(/[aeiou\d\W]/g, '')
```

I'm not sure you're supposed to strip all non-word-characters (\w) or all digits (\d). Consider re-reading the directions :)